pkgs/tools/system/fio/default.nix

```
@@ -16,6 +16,13 @@ stdenv.mkDerivation rec {
   buildInputs = [ python3 zlib ]
     ++ lib.optional (!stdenv.isDarwin) libaio;
 
+  # ./configure does not support autoconf-style --build=/--host=.
+  # We use CROSS_COMPILE= instead.
+  configurePlatforms = [ ];
+
+  makeFLags = lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+    "CROSS_COMPILE=${stdenv.hostPlatform.config}-"
```

Why not
```suggestion
    "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
```
which we may set unconditionally.